File: back/DevC/src/main/java/com/back/devc/domain/auth/dto/login/LoginResponse.java
package com.back.devc.domain.auth.dto.login;

import com.back.devc.domain.member.member.entity.MemberRole;
import com.back.devc.domain.member.member.entity.MemberStatus;

public record LoginResponse(
        Long userId,
        String email,
        String nickname,
        MemberRole role,
        MemberStatus status,
        String accessToken
) {
}

File: back/DevC/src/main/kotlin/com/back/devc/domain/auth/dto/login/LoginResponse.kt
package com.back.devc.domain.auth.dto.login

import com.back.devc.domain.member.member.entity.MemberRole
import com.back.devc.domain.member.member.entity.MemberStatus

data class LoginResponse(
    val userId: Long,
    val email: String,
    val nickname: String,
    val role: MemberRole,
    val status: MemberStatus,
    val accessToken: String
)
